### ISSUE TITLE
Shellcheck can't properly handle paths with spaces in them?

### ISSUE THREAD

**Issue by elsandosgrande** [koalaman/shellcheck#2517]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC1091
- My shellcheck version (`shellcheck --version` or "online"): 0.8.0
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)

It mentions paths and such, but it doesn't mention parsing errors due to spaces, so… 🤷🏼‍♂️??

- [ ] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

I don't see a way to edit multiple files on the site 😅

#### Here's a snippet or screenshot that shows the problem:

![image](https://user-images.githubusercontent.com/40576902/173253813-9b03d4c3-b8aa-4d95-9e74-fc9445d8080b.png)

```bash

#!/bin/bash

olive=1
source "/home/sandy/Projects/build scripts/linux/ocio-build.sh"

```

The other script is way too big to include here, but the relevant bit triggered by `test-script` is basically just `echo "It worked!"` if `olive` equals one.

#### Here's what shellcheck currently says:

```
$ shellcheck --color=always Projects/build\ scripts/test-script.sh

In Projects/build scripts/test-script.sh line 3:
olive=1
^---^ SC2034 (warning): olive appears unused. Verify use (or export if used externally).


In Projects/build scripts/test-script.sh line 4:
source "/home/sandy/Projects/build scripts/linux/ocio-build.sh"
       ^-- SC1091 (info): Not following: /home/sandy/Projects/build scripts/linux/ocio-build.sh was not specified as input (see shellcheck -x).

For more information:
  https://www.shellcheck.net/wiki/SC2034 -- olive appears unused. Verify use ...
  https://www.shellcheck.net/wiki/SC1091 -- Not following: /home/sandy/Projec...
$ shellcheck --version
ShellCheck - shell script analysis tool
version: 0.8.0
license: GNU General Public License, version 3
website: https://www.shellcheck.net
$ ls -lah "/home/sandy/Projects/build scripts/linux/ocio-build.sh"
-rwxrw-rw- 1 sandy sandy 38K  6月 12 22:54 '/home/sandy/Projects/build scripts/linux/ocio-build.sh'
```

#### Here's what I wanted or expected to see:

It following the path properly.

#### Additional information:

I tried using the `source` and `source-path` directives, but they just kept giving me parsing errors in VS Code unless I wrote a path without spaces, in which case the `(info)` line came back.

---

As far as I can tell, `vscode-shellcheck` uses the latest stable release of Shellcheck:

![image](https://user-images.githubusercontent.com/40576902/173254251-a5229ac5-8e6f-45be-a512-7a5d0502c410.png)

**Comment by koalaman** [comment]
You're right, this is simply not supported. Starting from 0.9.0 you will be able to quote the values with `'..'` or `".."` to allow file paths with spaces. Thanks!

**Comment by elsandosgrande** [comment]
Awesome! Also, sorry for seeing this so late 😅
